js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties

```diff
@@ -56,7 +56,7 @@ credentialsList=Lista de credenciales
 usermodel.clientRoleMapping.clientId.label=ID de cliente
 clientId=ID de cliente
 serviceProviderEntityId=ID de entidad del proveedor de servicios
-internationalizationHelp=Si está habilitado, puedes elegir qué ubicaciones admites para este reino y cuál es la ubicaci�n predeterminada.
+internationalizationHelp=Si está habilitado, puedes elegir qué ubicaciones admites para este reino y cuál es la ubicaci�n predeterminada.
```

Comment:
Can we remove the weird unicode here as well?
```suggestion
internationalizationHelp=Si está habilitado, puedes elegir qué ubicaciones admites para este reino y cuál es la ubicación predeterminada.
```

Reply:
Done, I apologise as i didn't see it